monai/networks/layers/factories.py

```diff
@@ -256,6 +256,13 @@ def swish_factory():
     return Swish
 
 
+@Act.factory_function("memswish")
```

Comment:
could you please add a test in https://github.com/Project-MONAI/MONAI/blob/master/tests/test_activations.py#L51

Reply:
This will be addressed in next commit.